swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, condition=0):
+        super().__init__("Clothing", condition)
```

Comment:
👍 good idea hardcoding "clothing" here since that will never change if you are creating an instance of Clothing!